packages/base-extension/codeGenerators/color.js

```diff
@@ -23,7 +23,14 @@ export const colorCodeGenerator = ({
         ? colorsParam
         : getResources({ context, useLinkedStyleguides: params.useLinkedStyleguides, key: "colors" });
 
-    const code = `${prefix}${allColors.map(c => generator.variable(c.getFormattedName("kebab"), new Color(c))).join(separator)}${suffix}`;
+    const code = `${prefix}${allColors.map(color => {
+        const colorName = (
+            color.getFormattedName
+                ? color.getFormattedName("kebab")
+                : color.name
+        );
```

Comment:
The older versions of the macOS app might pass color names that don't conform to the kebab-case scheme. 

So, instead of falling back to the name value, should we use an internal method (temporarily) in this extension to format names in case the extension models are not up-to-date in the app?

```js
function getFormattedName(color) {
  if (color.getFormattedName) {
    return color.getFormattedName("kebab");
  }

  // format name here
  let formattedName;
  ...

  return formattedName;
}
```

Reply:
After discussing this offline, this is a temporary issue that will be resolved automatically when the macOS app updates the extension model in the background. So, I think we don't need to address the issue in my comment above.